source/guide/manage-a-blog.html.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Manage a blog
+icon: book-open
+summary: 'How to customize your blog'
+---
+
+You can open your blog settings, by clicking on the "Edit" purple button in the top right part of its main page.
```

Comment:
it's only purple if the theme hasn't already been changed 😜

Reply:
The theme could even move it to another place, but I don't really know how to describe it then. ^^

Reply:
just leave out the colour, edit button is fine